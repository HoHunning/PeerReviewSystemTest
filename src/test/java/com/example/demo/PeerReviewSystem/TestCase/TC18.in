student 001 002 003 004
schoolStrategy A,100 B,90 C,80 D,70 E,60
designCriterion A1 criteria2_1.txt
averageCriterion A1