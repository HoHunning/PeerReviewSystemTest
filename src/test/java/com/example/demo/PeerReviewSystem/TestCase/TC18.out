Assignment: A1, Criterion: Criterion1, AvgScore: 0.0
Assignment: A1, Criterion: Criterion2, AvgScore: 0.0
